taxonium_web_client/src/Taxonium.jsx

```diff
@@ -69,13 +75,28 @@ function Taxonium({
     updateQuery,
     deckSize,
     xType,
+    settings
   });
 
-  //
+  // Treenome 
+  const [updateBrowserBounds, setUpdateBrowserBounds] = useState(false);
+  const [sidebarOpen, setSidebarOpen] = useState(true);
+  const toggleSidebar = () => {
+    // const tempView = view.viewState;
+
+    setSidebarOpen(!sidebarOpen);
+    setTimeout(() => {
+      window.dispatchEvent(new Event('resize'));
+    }, 100);
+
+  };
+
+  const browserState = useBrowserState(data, deckRef, view, settings)
```

Comment:
Nit: I wonder if we should call this e.g. `gBrowserState` throughout (or `genomeBrowserState` but maybe that's too long). Atm it always makes me think of the web-browser since that's such a common thing.

Reply:
Good call. Switched to `genomeBrowser` throughout

Reply:
Thanks!

Reply:
changed this again -- to `treenome` throughout for consistency (and shorter names)

Reply:
This is great - genome browser also had an implication it might apply just to the top panel